spec/builtin-process.test.sh

```diff
@@ -142,6 +143,135 @@ stat -c '%a' $TMP/umask-one $TMP/umask-two
 ## END
 ## stderr-json: ""
 
+#### umask -p
+umask -p | grep 'umask [0-9][0-9][0-9][0-9]'
+## status: 0
+## N-I dash/mksh/zsh status: 1
+
+#### umask -S
+# current mask as symbolic
+umask -S | grep 'u=[rwx]*,g=[rwx]*,o=[rwx]*' 
+## status: 0
+
+#### umask symbolic parsing
+
+umask 0000
```

Comment:
It would be nice to have some comments here explaining the syntax

Actually I think we should update `doc/ref/chap-builtin-cmd.md` with the syntax

Now it says it only supports octal, but we are adding symbolic input